tests/bson/bson-utcdatetimeinterface-002.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 MongoDB\BSON\UTCDateTimeInterface example with MongoDB\BSON\TypeWrapper
+--INI--
+date.timezone=UTC
```

Comment:
Unrelated to PHP 5.4. This is just something I noticed while testing the build against my PHP 5.5 ZTS environment, which didn't have `date.timezone` in its `php.ini`. All other test files that work with DateTime appear to define this in an `--INI--` block, so this is just something I forgot to include in https://github.com/mongodb/mongo-php-driver/commit/36487bc0a97421425ab1e067922aa7f078b888fc.